### ISSUE TITLE
Error while trying to type >

### ISSUE THREAD

**Issue by BugFixee** [PowerShell/PSReadLine#1719]
```
-----------------------------------------------------------------------
Last 200 Keys:
 c 1 d a a 4 7 c d 2 6 2 9 8 4 3 5 e f 2 d Enter
 > g Backspace Backspace g i t Space g r e p Space > Backspace c u r r e n t Space > Space o u t . t x t Enter
 l s Enter
 r m Space o u Tab Enter
 c d Space " Ctrl+v " Enter
 UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow UpArrow DownArrow Enter
 UpArrow UpArrow DownArrow DownArrow DownArrow DownArrow g r e p Space > Backspace - r n Space Ctrl+c g r e p Space - r n Space > Backspace " o u Backspace Backspace Backspace c u r r e n t C o d e Space > Space O u t . x t Enter
 . / o u Tab Enter
 e Backspace r m Space o u Tab Enter
 UpArrow UpArrow DownArrow UpArrow UpArrow Ctrl+LeftArrow LeftArrow LeftArrow LeftArrow Backspace o RightArrow RightArrow RightArrow t Enter
 . / o Tab Enter
 l s Enter
 c d Space . . Enter
 g r e p Space - r i n Space "

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
PS C:\Users\localhost\AppData\Local\Google\Chrome\User Data\Default\Extensions\ojhmphdkpgbibohbnpbfiefkgieacjmh\3.4.0_1>
```

**Comment by BugFixee** [comment]
I have a repro of this, if you want video / some other debug info.

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.4 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
